pkg/preflight/assets/host-preflights.yaml

```diff
@@ -276,3 +280,23 @@ spec:
               message: Port 6783 is open
           - warn:
               message: Unexpected port status
+    - tcpConnect:
+        checkName: "Kubernetes API TCP Connection Status"
+        collectorName: "Kubernetes API TCP Connection Status"
+        exclude: '{{ and .Installer.Spec.Kubernetes.Version .Installer.Spec.Kubernetes.MasterAddress .IsJoin | not }}'
+        outcomes:
+          - fail:
+              when: "connection-refused"
+              message: Connection to the Kubernetes API at address {{ .Installer.Spec.Kubernetes.MasterAddress }} was refused
+          - fail:
+              when: "connection-timeout"
+              message: Timed out connecting to the Kubernetes API at address {{ .Installer.Spec.Kubernetes.MasterAddress }}
+          - fail:
```

Comment:
This collector never attempts to bind

Reply:
I took this preflight from your host preflights google doc. should i be using something else as reference?

Reply:
That's the main reference